### PR TITLE
Multi grid output

### PR DESCRIPTION
**Pull request by pcarruscag**
## Proposed Changes
New output to visualize the coarse grids.
It is based on coloring nodes, which is a little difficult to visualize with Paraview since it interpolates to elements, anyway:
image

## PR Checklist
- [X] I am submitting my contribution to the develop branch.
- [X] My contribution generates no new compiler warnings (try with the '-Wall -Wextra -Wno-unused-parameter -Wno-empty-body' compiler flags, or simply --warnlevel=2 when using meson).
- [X] My contribution is commented and consistent with SU2 style.
- [X] I have added a test case that demonstrates my contribution, if necessary.
- [X] I have updated appropriate documentation (Tutorials, Docs Page, config_template.cpp) , if necessary.

### CODE REVIEW

**Comment by pcarruscag** [comment]
Last week folks mentioned, during the meeting, that this would be helpful, so I'll take it as approved.

File: Common/src/CConfig.cpp [hunk 1]
```diff
@@ -4996,9 +5004,9 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
   if (GetGasModel() == "ARGON") monoatomic = true;
 
   // This option is deprecated. After a grace period until 7.2.0 the usage warning should become an error.
-  if(OptionIsSet("CONV_CRITERIA")) {
-    cout << string("\n\nWARNING: CONV_CRITERIA is deprecated. SU2 will choose the criteria automatically based on the CONV_FIELD.\n") +
-            string("RESIDUAL for any RMS_* BGS_* value. CAUCHY for coefficients like DRAG etc.\n\n");
+  if(OptionIsSet("CONV_CRITERIA") && rank == MASTER_NODE) {
```
Comment (TobiKattmann) [inline Common/src/CConfig.cpp:5007]: makes it less annoying, sure, but that was the purpose to some degree :D (totally did that on purpose ... 🤥 )

File: SU2_CFD/src/output/CFlowOutput.cpp [hunk 2]
```diff
@@ -2735,6 +2735,41 @@ bool CFlowOutput::WriteVolume_Output(CConfig *config, unsigned long Iter, bool f
   return false || force_writing;
 }
 
+void CFlowOutput::AddCommonFVMOutputs(const CConfig *config) {
```
Comment (TobiKattmann) [inline SU2_CFD/src/output/CFlowOutput.cpp:2738]: 👍 Special reason to not move `COORD-XYZ`  here?

**Review by TobiKattmann — COMMENTED** [review]
💐 Thanks, also for the little stuff around

File: SU2_CFD/src/output/CFlowOutput.cpp (hunk 2 above)
Comment (pcarruscag) [reply SU2_CFD/src/output/CFlowOutput.cpp:2738]: Order matters I think.

File: Common/src/CConfig.cpp (hunk 1 above)
Comment (pcarruscag) [reply Common/src/CConfig.cpp:5007]: Yeah you can't miss the message if you run 1000 cores

File: SU2_CFD/src/output/CFlowOutput.cpp (hunk 2 above)
Comment (TobiKattmann) [reply SU2_CFD/src/output/CFlowOutput.cpp:2738]: yep it does ... it needs to be the first, at least for paraview output

**Pull request merged**